aerospike/assets/configuration/spec.yaml

```diff
@@ -487,5 +362,141 @@ files:
           - dc_ship_destination_error
           - dc_ship_source_error
           - dc_ship_success
+  - template: instances
+    options:
+    - template: instances/openmetrics
+      overrides:
+        openmetrics_endpoint.enabled: true
+        openmetrics_endpoint.required: false
+        openmetrics_endpoint.value.example: http://127.0.0.1:9145/metrics
+        openmetrics_endpoint.display_priority: 1
+        username.description: |
+          The username to use if services are behind basic or digest auth or db user to authenticate (Legacy).
+        password.description: |
+          The password to use if services are behind basic or NTLM auth, or the password for db username (Legacy).
+        timeout.description: |
+          The timeout for accessing services
+          Read timeout in seconds when using Legacy.
+    - name: host
```

Comment:
Let's hide anything legacy so new customers only use the updated OpenMetrics implementation.